openamp/libmetal.defs

```diff
@@ -79,6 +79,7 @@ libmetal.zip:
 	$(Q) mv libmetal-$(VERSION) libmetal
 	$(Q) patch -p0 < 0001-libmetal-add-metal_list_for_each_safe-support.patch
 	$(Q) patch -p0 < 0002-libmetal-nuttx-io.c-align-access-when-read-write-siz.patch
+	$(Q) patch -p0 < 0003-libmetal-nuttx-io.c-Fix-void-pointer-arithmetic-in-a.patch
```

Comment:
@jlaitine @CV-Bowen let's send the patch to OpenAMP community too.

Reply:
Sure, the original patch is in this PR: https://github.com/OpenAMP/libmetal/pull/279 ; this is not merged in the openamp libmetal yet. So obviously I can't provide the fix for that patch either :)

@CV-Bowen feel free to add my patch to the same PR, or squash the fix into your original patch. I am not sure what is the plan forward with that PR.

Reply:
@xiaoxiang781216 @jlaitine Thanks, I will update https://github.com/OpenAMP/libmetal/pull/279 later today.